__tests__/app.test.js

```diff
@@ -121,4 +120,42 @@ describe("GET: /api/topics", () => {
             expect(body.msg).toBe('Not Found')
         })
     })
- })
\ No newline at end of file
+ })
+
+ describe("GET: /api/articles", () => {
+    test("GET:200 returns an array of article objects", () => {
+        return request(app).get("/api/articles")
+        .expect(200)
+        .then(({body}) => {
+            console.log(body.articles)
+            expect(Array.isArray(body.articles)).toBe(true)
+            expect(body.articles.length).not.toBe(0)
```

Comment:
No need to check the length is not 0, but it would be nice to check the actual length